corehq/apps/importer/tests.py

```diff
@@ -334,3 +335,27 @@ def testLocationOwner(self):
         error_message = ImportErrors.InvalidOwnerId
         self.assertIn(error_message, res['errors'])
         self.assertEqual(res['errors'][error_message]['rows'], [5])
+
+
+class ImporterUtilsTest(SimpleTestCase):
```

Comment:
i didn't bother adding an explicit test for location owners because it's already tested by the test that failed